src/web/app/pages-instructor/instructor-session-individual-extension-page/instructor-session-individual-extension-page.component.ts

```diff
@@ -0,0 +1,528 @@
+  sortInstructorOrder: SortOrder = SortOrder.DESC;
+
+  isAllStudentsSelected: boolean = false;
+  isAllInstructorsSelected: boolean = false;
+
+  courseId: string = '';
+  courseName: string = '';
+  feedbackSessionName: string = '';
+
+  feedbackSessionEndingTimestamp: number = 0;
+  feedbackSessionTimeZone: string = 'UTC';
+
+  feedbackSessionDetails: FeedbackSessionBasicRequest = {
+    instructions: '',
+    submissionStartTimestamp: 0,
+    submissionEndTimestamp: 0,
+    gracePeriod: 0,
+    sessionVisibleSetting: SessionVisibleSetting.AT_OPEN,
+    customSessionVisibleTimestamp: 0,
+    responseVisibleSetting: ResponseVisibleSetting.CUSTOM,
+    customResponseVisibleTimestamp: 0,
+    isClosingEmailEnabled: false,
+    isPublishedEmailEnabled: false,
+  };
+
+  columnsData: ColumnData[] = [];
+  rowsData: SortableTableCellData[][] = [];
+
+  studentsOfCourse: StudentExtensionTableColumnModel[] = [];
+  instructorsOfCourse: InstructorExtensionTableColumnModel[] = [];
+  studentDeadlines: Record<string, number> = {};
+  instructorDeadlines: Record<string, number> = {};
+
+  isLoadingAllStudents: boolean = true;
+  hasLoadedAllStudentsFailed: boolean = false;
+  isLoadingAllInstructors: boolean = true;
+  hasLoadedAllInstructorsFailed: boolean = false;
+  isLoadingFeedbackSession: boolean = true;
+  hasLoadingFeedbackSessionFailed: boolean = false;
+  isSubmittingDeadlines: boolean = false;
+
+  ngOnInit(): void {
+    this.route.queryParams.subscribe((queryParams: any) => {
+      this.courseId = queryParams.courseid;
+      this.feedbackSessionName = queryParams.fsname;
+      this.loadFeedbackSessionAndIndividuals();
+    });
+  }
+
+  constructor(
+    private statusMessageService: StatusMessageService,
+    private feedbackSessionsService: FeedbackSessionsService,
+    private studentService: StudentService,
+    private ngbModal: NgbModal,
+    private route: ActivatedRoute,
+    private courseService: CourseService,
+    private tableComparatorService: TableComparatorService,
+    private instructorService: InstructorService,
+  ) {}
+
+  /**
+   * Loads a feedback session and individuals
+   */
+  loadFeedbackSessionAndIndividuals(): void {
+    this.resetTables();
+    this.isLoadingAllStudents = true;
+    this.hasLoadedAllStudentsFailed = false;
+    this.isLoadingFeedbackSession = true;
+    this.hasLoadingFeedbackSessionFailed = false;
+    this.isLoadingAllInstructors = true;
+    this.hasLoadedAllStudentsFailed = false;
+    forkJoin([
+      this.courseService.getCourseAsInstructor(this.courseId),
+      this.feedbackSessionsService.getFeedbackSession({
+        courseId: this.courseId,
+        feedbackSessionName: this.feedbackSessionName,
+        intent: Intent.FULL_DETAIL,
+      }),
+    ])
+      .pipe(finalize(() => {
+        this.isLoadingFeedbackSession = false;
+        this.isLoadingAllStudents = false;
+        this.isLoadingAllInstructors = false;
+      }))
+      .subscribe(
+        (value: any[]) => {
```

Comment:
Is it possible to make `value`'s type here `[Course, FeedbackSession]`? If so, we can simply destructure it instead of line 138 and line 140